src/lib/config.ts

```diff
@@ -87,7 +89,7 @@ export class LauncherConfig implements Serializable<LauncherConfig> {
     this._loaded = true;
   }
 
-  private async saveConfigToFile() {
```

Comment:
Keep this private.

Reply:
I think you should, at the end of this file(src/lib/config.ts) there are 3 public setters(lines [226 ](https://github.com/open-goal/launcher/blob/e5f8ce31f5ceac292b081f63556568b20e421a21/src/lib/config.ts#L226) to 252) and thats the only place in the code where saveConfigToFile() is used, so to change the language you should create a new setter and use that.

Reply:
Yea, did it like you said @xsm2